src/main/groovy/com/okta/test/mock/application/ApplicationTestRunner.groovy

```diff
@@ -83,7 +87,7 @@ abstract class ApplicationTestRunner extends HttpMock {
 
         startMockServer()
         app.start()
-
+        
```

Comment:
Please make sure that trailing whitespace is removed (use IDE configuration)

Reply:
Done